pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -257,6 +258,18 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, nil
 	}
 
+	// Only allow live migration in managed clusters excluding hypershift clusters
+	isStandaloneManagedCluster, err := util.IsStandaloneManagedCluster(ctx, r.client)
```

Comment:
Wouldn't that whole check better fit into `isMigrationChangeSafe`?
https://github.com/openshift/cluster-network-operator/blob/a5edddf194e70a8ab01a44dbc4e3d8214a773f10/pkg/network/render.go#L673

Reply:
I think validate is better. Move it there.